packages/mobile/capacitor/capacitorApi.ts

```diff
@@ -316,10 +328,10 @@ export const CapacitorApi: IPlatform = {
         if (os === 'ios') {
             void (await SecureFilesystemAccess.allowAccess())
         }
-        void (await SecureFilesystemAccess.saveRecoveryKit({
-            selectedPath: `${selected}/recovery-kit.pdf`,
-            fromRelativePath: '/assets/docs/recovery-kit.pdf',
-        }))
+        // void (await SecureFilesystemAccess.saveRecoveryKit({
```

Comment:
should this be commented out?

Reply:
`saveRecoveryKit` was removed from `SecureFilesystemAccess` and we dont use it on mobile atm, without this commented out the app cant compile